### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by morgabm** [LizardByte/community#476] [Sunshine] [answered]
Is there a good place to start? Most I have found is [CONTRIBUTING.rst](https://github.com/LizardByte/.github/blob/e0cd360454c18a75f91aff1390798d30835a31ca/CONTRIBUTING.rst) on the group page. 

Is there no wiki documenting architecture, design decisions, quick start, etc? If not, I might be willing to begin contributing and document my process in an effort to convert said docs to such a resource. Is there a community forum such as slack or discord?

**Comment by ReenigneArcher** [comment, marked as answer]
Those are general guidelines for any of our repos, but there is some more specific info in the Sunshine docs. Not every single detail is documented as things can change pretty quickly. https://docs.lizardbyte.dev/projects/sunshine/en/master/contributing/contributing.html

**Reply by morgabm** [reply]
Thanks for the docs link. I am not actually a user of the software but the domain itself is interesting and adjacent to my professional duties hence my interest. I will start by reading up and possibly trying out the software.

**Reply by ReenigneArcher** [reply]
You're welcome! Let me know if you have any questions. Probably easier to do that via our discord server, https://app.lizardbyte.dev/discord

I'm not sure what specific areas you are interested in. There's definitely a lot of different specialties that you could focus on in this one project, but...

One of my upcoming goals is to complete the source code documentation. I've recently been spending a lot of time improving the state of the doxygen builds, but the main thing that remains is https://github.com/LizardByte/Sunshine/blob/7e9a2dd621d803a58d460e698e897c3602bf07e2/docs/Doxyfile#L79-L80

Once that is enabled, the docs builds will actually error if anything is undocumented, which will be nice for CI/CD of PRs.

There's also a basic roadmap (GitHub project) here: https://github.com/orgs/LizardByte/projects/13
